api/src/main/resources/workbench.yaml

```diff
@@ -888,6 +888,228 @@ paths:
           schema:
             $ref: "#/definitions/CohortSummaryListResponse"
 
+
+  # Data Browser #######################################################################
+  /api/v1/databrowser/search-concepts:
```

Comment:
One question: do we expect that researchers will be able to toggle the CDR version in the data browser?

If there's any chance they will, there ought to be a parameter either in the path or query for all of these indicating what CDR version to use.

We could add that later -- and I think we *definitely* need to do something like this in future for the cohort builder API. @blrubenstein @johnstill   For now, this is fine.